docs(about us): update contributors list and numbers (#22559)
renovatebot/renovate: docs/usage/about-us.md

- [@secustor](https://github.com/secustor) worked on Terraform and Helm
- [@fgreinacher](https://github.com/fgreinacher) worked on NuGet
- [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo
- [@HonkingGoose](https://github.com/HonkingGoose) works on the documentation and helps manage the community

### Valuable contributions

We want to highlight these valuable contributions, even if they are one-offs.
Some features made a lot of people happy, and efficient!

- [@ikesyo](https://github.com/ikesyo) regularly helpful
- [@astellingwerf](https://github.com/astellingwerf) reviews PRs
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
- [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
- [@setchy](https://github.com/setchy) focused on Bitbucket Cloud and replacement features
- [@cgrindel](https://github.com/cgrindel) created the `bazel-module` manager

> - [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored our code and ...

Author: @RahulGautamSingh I see you refactored a lot of our code, but don't know what to highlight extra (special feature/good bugfix).

Reviewer: I think the highlight of my commits will be perf improvements after reduced cloning during update and onboarding(ongoing).

Author: ```suggestion
- [@RahulGautamSingh](https://github.com/RahulGautamSingh) refactored a lot of code and worked on performance improvements like reduced cloning during updates and onboarding
```

How about this?

> - [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) ...

Author: @Gabriel-Ladzaretti Let me know what you'd like to be credited for here. I see you landed enough commits to be included in this list (if you want). 😉

Reviewer 2: Thanks a lot for this :), 
I really liked working on some features such as s3 backed repo cache, implementing our own process spawn and the config migration PR but there is no single focus point for me. so i guess something like "regularly helpful" might fit ? ¯\\_(ツ)_/¯

Author: I see the maintainer pushed a commit that describes your work. Let me know if you like it. 😉

> - [@Churro](https://github.com/Churro) worked on the `gradle` manager

Author: @Churro please check if you're happy with this credit on our About Us page. 😉

Reviewer 3: Thanks for the credits!

Looking at my PRs, I'd suggest the following addition:

```suggestion
- [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities
```

> - [@Churro](https://github.com/Churro) worked on the `gradle` manager

Reviewer 3: Thanks for the credits! 

Looking at my PRs, I'd suggest the following addition: 
```suggestion
- [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities
```

> - [@Churro](https://github.com/Churro) worked on the `gradle` manager and added support for OSV vulnerabilities

Reviewer 4: I think @Churro should be added to the regular section

> - [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) ...

Reviewer 4: ```suggestion
- [@Gabriel-Ladzaretti](https://github.com/Gabriel-Ladzaretti) S3 repo cache, child process refactoring, others
```
